src/style.css

```diff
@@ -0,0 +1,76 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px ;
+  height: 60px;
```

Comment:
what happens if you remove this height?

Reply:
I have removed height in main-container and have included line-height: 60px in nav__link.

Reply:
I have removed height and have setted line-height 60 px. in nav__link.